src/components/Landing.jsx

```diff
@@ -1,9 +1,25 @@
+import landingbg from "../../public/landingbg.svg";
+import landingtitle from "../../public/landingtitle.svg";
+import Image from "next/image";
 import React from "react";
 
 const Landing = () => {
   return (
-    <div>
-      <p>Landing</p>
+    <div className="flex justify-center px-4 py-4">
+      <Image
+        src={landingbg.src}
+        alt="Landing Picture"
+        width={1000}
+        height={1000}
```

Comment:
try className"w-5/6" see how that looks but make sure to remove width and height {}